AngularApp/projects/app-service-diagnostics/src/app/shared/components/tools/network-checks/network-check-flows/functionsFlow.js

```diff
@@ -543,13 +557,153 @@ async function validateConnectionStringAsync(propertyName, connectionString, typ
     }
     return subChecks;
 }
+async function validateConnectionViaAppSetting(propertyName, connectionString, type, diagProvider, failureDetailsMarkdown = undefined,entityName = undefined) {

```

Comment:
Why do you have the parameter "connectionString" in this method?  It's never used and does not make sense in a function that operates based on the AppSetting that it will look up.